src/protocolgame.cpp

```diff
@@ -2778,10 +2778,21 @@ void ProtocolGame::sendHouseWindow(uint32_t windowTextId, const std::string& tex
 
 void ProtocolGame::sendOutfitWindow()
 {
+	const auto& outfits = Outfits::getInstance().getOutfits(player->getSex());
+	if (outfits.size() == 0) {
+		return;
+	}
+
 	NetworkMessage msg;
 	msg.addByte(0xC8);
 
 	Outfit_t currentOutfit = player->getDefaultOutfit();
+	if (currentOutfit.lookType == 0) {
```

Comment:
why this? If you have looktype of item then there should be no problem with sending current looktype even if you look as item, cause item looktype is also stored in Outfit_t

Reply:
1. tfs doesn't support adding item outfits, but you can set one through lua
2. I'm not even sure if item clientId can be sent in outfit window packet without crashing the client
3. very likely. At this moment I don't have enough time to reproduce the full outfit window packet.

Reply:
OTClient supports displaying item id in outfit window, might be good to add a client check before forcing random outfit.

Anyway, I believe the `AddOutfit` should not be called within this method (unless it's an OTClient), just write the bytes like the client expects them.

Reply:
I'm going to guess that the Outfit window doesn't support "item" look, but it would be nice to know for sure.

Reply:
It does not: https://github.com/Cavitt/TibiaFlashClient/blob/master/scripts/tibia/network/Communication.as#L5950